### ISSUE TITLE
Why is fullscreen mode disabled for IE11 when documents are embedded?

### ISSUE THREAD

**Issue by Ada323** [mozilla/pdf.js#9585]
I asked about fullscreen mode support for IE11 in https://github.com/mozilla/pdf.js/issues/9572, and was told that it is disabled when documents are embedded. Can you please give me more information about why this is necessary? I'd like to try to fix the issue (if possible) and it would be helpful to know what bugs to look out for that caused fullscreen mode to be disabled for embedded documents IE11. In my initial exploration of this issue, I didn't see any obvious problems when I enabled fullscreen mode for some documents rendered in an iframe in IE11. Thank you!

**Comment by Snuffleupagus** [comment]
It's disabled because of bugs in Internet Explorer; please refer to issue #4711.

**Comment by Ada323** [comment]
Thank you for the quick response. That bug was reported and addressed in 2014 - it's possible that it's been fixed on Internet Explorer's end in the last few years. I'm going to continue investigating but I've tested it out on several files (all rendered within iframes) and I have not found any issues with using fullscreen mode (the only change I made to test this is commenting out the if statement where fullscreen is disabled for IE). I have been able to successfully navigate between pages with arrow keys, clicking, and scroll wheel. I will let you know if I notice any problems but you might consider revisiting that earlier issue.
